engines/director/lingo/lingo-builtins.cpp

```diff
@@ -672,6 +672,37 @@ void LB::b_deleteAt(int nargs) {
 	}
 }
 
+void LB::b_deleteOne(int nargs) {
+	Datum indexD = g_lingo->pop();
+	Datum list = g_lingo->pop();
+	TYPECHECK3(indexD, INT, FLOAT, SYMBOL);
+	TYPECHECK2(list, ARRAY, PARRAY);
+
+	switch (list.type) {
+	case ARRAY: {
+		g_lingo->push(list);
+		g_lingo->push(indexD);
+		b_getPos(nargs);
```

Comment:
I changed the code after the stream. getPos(list, value) returns the index of the first copy of value in the list, which then gets passed to remove_at.

The variable name I copypastad from the old function so it isn't clear, I've since fixed it.